data/json/recipes/recipe_armor.json

```diff
@@ -9398,7 +9398,33 @@
       [ "felt_patch", 8 ]
     ]
   ]
- },
+ },{
+  "type" : "recipe",
+  "result": "makeshift_knapsack",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_STORAGE",
+  "skill_used": "survival",
+  "difficulty": 0,
```

Comment:
I see no reason for this to be tailoring. I say survival 1, like for instance stone pot.